guides/common/modules/ref_system-requirements.adoc

```diff
@@ -44,6 +44,12 @@ endif::[]
 If your territory is USA and your language is English, set `en_US.utf-8` as the system-wide locale settings.
 For more information about configuring system locale in {EL}, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring System Locale guide].
 
+ifdef::satellite[]
+Your {Project} must have the {SatelliteSub} manifest in your Customer Portal.
+Also, the {Project} must have {project-context}-{smart-proxy-context}-6.x repository enabled and synced.
+To create, manage, and export a Red{nbsp}Hat Subscription Manifest in the Customer Portal, see https://access.redhat.com/documentation/en-us/subscription_central/2021/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
```

Comment:
When I clicked on this link, I was forwarded to https://access.redhat.com/documentation/en-us/subscription_central/2023